File: DACN/Models/Report.cs
using System;
using System.ComponentModel.DataAnnotations;

using System.ComponentModel.DataAnnotations.Schema;
namespace Models
{
    public class Report
    {
        [Key]
        public int Id { get; set; }
    [Column("created_at")]
    public DateTime? CreatedAt { get; set; }
    public string? Type { get; set; }
    [Column("file_path")]
    public string? FilePath { get; set; }
    [Column("created_by")]
    public int CreatedBy { get; set; }
    }
}

File: DACN/backend_src/app/models/report.py
from app.database import Base
from sqlalchemy import Column, Integer, String, DateTime, ForeignKey
from sqlalchemy.orm import relationship

class Report(Base):
    __tablename__ = "reports"
    id = Column(Integer, primary_key=True, index=True)
    created_at = Column(DateTime, nullable=False)
    type = Column(String(50), nullable=False)
    file_path = Column(String(255))
    created_by = Column(Integer, ForeignKey("users.id", ondelete="SET NULL"), nullable=True)
    user = relationship("User")
